tests/hw3/test_task1.py

```diff
@@ -0,0 +1,30 @@
+from io import StringIO
+from typing import Callable
+
+import pytest
+
+from hw3.task1.hw1 import cache
+
+
+def f():
```

Comment:
Why not just import the function from the completed task file?

Reply:
done